media/css/etc/firefox/retention/thank-you-a.scss

```diff
@@ -0,0 +1,216 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../pebbles/includes/lib';
+@import '../../../pebbles/components/newsletter';
+
+body {
+    border-top: 0;
+}
+
+html {
+    font-size: 100%;
+}
+
+@font-face {
+    font-family: antonioregular;
+    src: url('/media/fonts/fx-lifestyle/antonio-regular-webfont.woff2') format('woff2'),
+         url('/media/fonts/fx-lifestyle/antonio-regular-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: antoniobold;
+    src: url('/media/fonts/fx-lifestyle/antonio-bold-webfont.woff2') format('woff2'),
+         url('/media/fonts/fx-lifestyle/antonio-bold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Open Sans';
+    font-weight: normal;
+    font-style: normal;
+    src: url('/media/fonts/opensans-regular.woff2') format('woff2'),
+         url('/media/fonts/opensans-regular.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'Open Sans';
+    font-weight: bold;
+    font-style: normal;
+    src: url('/media/fonts/opensans-semibold.woff2') format('woff2'),
+         url('/media/fonts/opensans-semibold.woff') format('woff');
+}
+
+main {
+    overflow: hidden;
+    position: relative;
+    padding: 0 10px;
+    background: #45A5B3;
+    color: #FFFFFF;
+
+    #confetti-canvas {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        height: 100%;
+        width: 100%;
+        max-width: 2200px;
+        left: 0;
+        right: 0;
+        margin: 0 auto;
+    }
+
+    .thank-you-header {
+        font-family: antoniobold;
+        text-transform: uppercase;
+        font-size: 6.25rem;
+        line-height: 6.25rem;
+        div {
+          color: #F8EB57;
+          text-shadow: -5px 0 0 rgb(215, 102, 40), -8px 0 0 rgba(32, 35, 36, 0.3);
+        }
+        div:first-child {
+          color: #FFFFFF;
+          text-shadow: -5px 0 0 rgb(148, 209, 175), -8px 0 0 rgba(32, 35, 36, 0.3);
+        }
+    }
+
+    p {
+        font-size: 1.3rem;
+        font-family: "Open Sans";
+        font-size: 1.75rem;
+        font-weight: bold;
+    }
+
+    @media #{$mq-phone-wide} {
+        padding: 0 30px;
+    }
+
+    @media #{$mq-tablet} {
+        padding: 0;
+    }
+    .thank-you-actions-container {
+        display: flex;
+        flex-wrap: wrap;
```

Comment:
Forgot to double check this, but after a quick look, we're probably fine as is. IE 11 seems to be fine, and while IE10 doesn't quite display it the same, I don't think it's broken, just lacking an enhancement.